firmware/Cargo.toml

```
@@ -33,7 +34,10 @@ default = ["mcu-esp32c3", "imu-mpu6050", "log-rtt"]
 
 # todo: actually support another microcontroller
 # Supported microcontrollers
-mcu-esp32c3 = ["defmt_esp_println?/esp32c3"]
+mcu-esp32c3 = ["defmt_esp_println?/esp32c3", "esp-wifi", "esp-backtrace/esp32c3"]
+
+# Wi-fi dependencies
+esp-wifi = ["dep:esp-wifi"]
```

stay consistent with `category-variant` features
```suggestion
net-wifi = ["dep:esp-wifi"]
```